### PR TITLE
Fix building without sandboxing

### PR DESCRIPTION
**Pull request by keith**
Since 466c32c70f6262f43eac06ad5e9dc2cbecbba228 any changes you made
while testing rules_foreign_cc, or changes to those rules, would not
invalidate the CMakeCache.txt and lead to build issues. It wasn't the
case before that because new temp dirs were used each time.

### CODE REVIEW

**Comment by keith** [comment]
cc @attilaolah @scele

File: foreign_cc/private/framework/toolchains/windows_commands.bzl [hunk 1]
```diff
@@ -41,6 +41,9 @@ def disable_tracing():
 def mkdirs(path):
     return "mkdir -p " + path
 
+def rm_rf(path):
+    return "rm -rf " + path
```
Comment (keith) [inline foreign_cc/private/framework/toolchains/windows_commands.bzl:45]: Is this valid for windows? I see some places using this without going through this abstraction so maybe it is?

Comment (jsharpe) [reply foreign_cc/private/framework/toolchains/windows_commands.bzl:45]: Yes - its running in a msys2 environment so rm -rf is fine.

**Review by jsharpe — APPROVED** [review]
Thanks for finding this issue!

**Pull request merged**